Mathlib/FieldTheory/ConjAct.lean

```diff
@@ -0,0 +1,22 @@
+/-
+Copyright (c) 2022 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import Mathlib.Algebra.Field.Defs
+import Mathlib.GroupTheory.GroupAction.ConjAct
+
+/-!
+# Conjugation action on fields
+-/
+
+namespace ConjAct
+
+variable {K : Type*} [DivisionRing K]
```

Comment:
It seems a little arbitrary to me to say that `DivisionRing` is too heavy, but `GroupWithZero` and `Semiring` are fine.

Reply:
They are *not* fine, but they are much harder imports to remove. I would need a lot more prerequisite splits/API rewrites before I can remove even `Semiring`

Reply:
Could you not split this file along the Group / not-just-group line, and deal with the second one being too heavy later?

Reply:
Sure. Any suggestion for the name of that second file? The same as I have now?

Reply:
How about `Mathlib.Algebra.GroupWithZero.Action.ConjAct`?

Note you'll need to update the docstring in `ConjAct` to point out that some content is elsewhere.

Reply:
Maybe I'll make it `Algebra.Field...` instead, since I am trying to keep the invariant that `Algebra.Group` doesn't know about `GroupWithZero`, `Algebra.GroupWithZero` doesn't know about `Semiring`, etc...